### ISSUE TITLE
Maven repository

### ISSUE THREAD

**Issue by jmvivo** [javaparser/javaparser#17]
Hello,

Is there the library (the current version 1.0.9) deployed on any maven repository?
If not, is there any schedule about it?

Thanks.

Regards
Chema.

**Comment by stepinto** [comment]
+1 for need of a release on maven repo

**Comment by arouel** [comment]
+1

I can help to get all changes in pom.xml etc. to be valid for upload in central repository. Interested?

**Comment by matozoid** [comment]
I've tried contacting the original author before, but he has not supplied information on how he did the releases. Who wants to pick this up?

**Comment by arouel** [comment]
I would do that. In current state, the coordinates of this release should be:

``` xml
        <groupId>com.github.matozoid</groupId>
        <artifactId>javaparser</artifactId>
        <version>1.0.9</version>
```

Is this okay for us?

An alternative is to build an organization on GitHub to host the _javaparser_ project. Then Sonatype accepts also

``` xml
        <groupId>com.github.javaparser</groupId>
        <artifactId>javaparser</artifactId>
        <version>1.0.9</version>
```

**Comment by matozoid** [comment]
I feel weird having my nick on it (matozoid,) since the only thing I do is keep a central place for patches. So either the com.github one, or consider keeping the old one (even though it's not on google code anymore:)

``` xml
<dependency>
    <groupId>com.google.code.javaparser</groupId>
    <artifactId>javaparser</artifactId>
    <version>1.0.9</version>
</dependency>
```

Oh, and please supply a short description of how to do a release, so anyone can do it next time ;)

**Comment by arouel** [comment]
We can try to upload it under the original coordinates.

Rules that should apply to us:
_We encourage (require) people to use a groupId they control by project or domain to name their artifacts. We have to be careful though how to apply this rule when people are trying to upload artifacts they don't control. The first inclination is for people to rename their artifacts to be under a groupId they do control. This is bad because 1) other people who would like to use these artifacts won't find them buried in your groupId and 2) classpath conflicts if the same artifact is uploaded under another groupId._
https://docs.sonatype.org/display/Repository/Choosing+your+Coordinates

**Comment by matozoid** [comment]
Go ahead before :)

**Comment by arouel** [comment]
@matozoid I preparing the POM to be able to upload artifacts with maven-release-plugin. During this i've mentioned compiling issues, because the grammar has UTF-8 characters (like ù) and these chars will not be translated correctly in the generates classes. This issue occur only on OSX. Tests under Linux are successful. On which platform do you work?

**Comment by arouel** [comment]
I've released javaparser 1.0.9 successfully to Central Repository and open a pull request to merge my changes into this master.

http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22javaparser%22
